Sources/Misc/SystemInfo.swift

```diff
@@ -175,6 +183,10 @@ class SystemInfo {
         self.dangerousSettings = dangerousSettings ?? DangerousSettings()
         self.clock = clock
         self.preferredLocalesProvider = preferredLocalesProvider
+
+        self.isApplicationBackgrounded { isAppBackgrounded in
```

Comment:
There may be a short span of time where the value does not reflect the correct value until this is executed... That's the main reason I didn't want to start changing usages of the existing `isAppBackgrounded` API everywhere. I think `isAppBackgroundedState` may be used for less critical code paths. There are some places we could potentially change it, but I wanted to minimize the changes in this PR, which gets big with the snapshot changes already.